features/add_a_book.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Add book to a library
+
+
+    Scenario: Admin adds new book to the library giving correct data
+
+        Given Admin is logged in
```

Comment:
Podobno można takie coś obejść, można ustawić na początku danego feature, albo w pierwszym kroku login as admin i tam ukryć logowanie (tak mi Paulina napisała). Ja u siebie ustawiłem jako zasadę Rule, ale to nie jest błąd tylko się czepiam ;d https://cucumber.io/docs/gherkin/reference/